src/openrct2/Cheats.cpp

```diff
@@ -384,19 +375,32 @@ static void cheat_remove_all_guests()
             {
                 vehicle = GET_VEHICLE(spriteIndex);
 
-                vehicle->num_peeps = 0;
-                vehicle->next_free_seat = 0;
+                for (int i = 0; i < vehicle->num_peeps; i++) {
```

Comment:
Please use `size_t`, rather than `int`. Could you also put the brace on the next line?